utils/Makefile

```diff
@@ -0,0 +1,20 @@
+all: all_individuals.csv, all_organizations.csv, all_transactions.csv
+
+#set this up as a pipeline, check how far along Adil is on this
+output deduplicated_individuals: all_individuals.csv
+	python deduplication_pipeline
+
+#likewise here
+output deduplicated_organizations: all_organizations.csv
+	python deduplication_pipeline
+
+
+output classified_individuals: all_individuals.csv
```

Comment:
there should be 1 pipeline that is run -- adil's pipeline will preprocess, dedupe, classify, and then build the network